gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/IncrementGuard.kt

```diff
@@ -34,7 +34,19 @@ class IncrementGuard : Plugin<Project> {
         const val taskName = "checkVersionIncrement"
     }
 
+    /**
+     * Adds the [CheckVersionIncrement] task to the project.
+     *
+     * Only adds the check if the project is built on Travis CI and the job is a pull request.
+     *
+     * The task will never run outside of Travis CI or when building individual branches. This is
+     * done to prevent unexpected CI fails when re-building `master` multiple times, creating git
+     * tags, and in other cases that go outside of the "usual" development cycle.
+     */
     override fun apply(target: Project) {
+        if (!isTravisPullRequest()) {
+            return
```

Comment:
Is it possible to add some logs here? To make it really explicit that we're not adding the task on purpose.